README.md

```diff
@@ -1,17 +1,17 @@
 <a title="https://jwallet.github.io/spy-spotify/" href="https://jwallet.github.io/spy-spotify/"><img alt="Spotify Logo" src="https://user-images.githubusercontent.com/23088305/29906214-6daad21c-8de1-11e7-80f5-ef6791cc7825.png" /></a>
 
-Spytify is a Spotify recorder for Windows which records audio from your sound card, ensuring no loss in quality of songs recorded from Spotify, without recording or playing its ads. It automatically splits songs into separate tracks and records straight to MP3 with media infos and album cover, meaning you can start enjoying your music offline.
+Spytify is a Spotify recorder for Windows which records Spotify audio without recording or playing ads, ensuring no loss in sound quality. It automatically splits songs into separate tracks and records to MP3 with media metadata, meaning you can start easily enjoying your music offline.
```

Comment:
i think it's important to say it records from the sound card, so people will know that system sounds can be recorded as well if they are not mute